src/explore-education-statistics-frontend/src/components/PrintThisPageAnalytics.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+
+const PrintThisPage = () => {
```

Comment:
Why have we created a new component here? Given we augmented `Link` with `AnalyticsProps`, it feels like we should do a similar thing here as well